lib/logitech_receiver/settings.py

```diff
@@ -211,6 +211,33 @@ def write(self, device, data_bytes):
 		assert self.feature is not None
 		return device.feature_request(self.feature, self.write_fnid, data_bytes)
 
+class NewFeatureRW(object):
+
+	__slots__ = ('feature', 'read_fnid', 'write_fnid', 'prefix', 'key')
+
+	kind = _NamedInt(0x02, 'feature')
+	default_read_fnid = 0x10
+	default_write_fnid = 0x20
+
+	def __init__(self, feature, prefix, key, read_fnid=default_read_fnid, write_fnid=default_write_fnid):
+		assert isinstance(feature, _NamedInt)
+		self.prefix = prefix & 0xFFFF
+		self.key = key
+		self.feature = feature
+		self.read_fnid = read_fnid
+		self.write_fnid = write_fnid
+
+	def read(self, device):
+		# Reading requires passing some parameters
+		value = device.feature_request(self.feature, self.write_fnid,
+					((self.prefix << 16) + 0xFFFF).to_bytes(4, 'big'))
```

Comment:
Why use self.write_rnid here instead of self.read_rnid?

Reply:
Sorry, I've just fixed that.